templates/web/oxfordshire/index-steps.html

```diff
@@ -1,3 +1,9 @@
+<h2>[% loc('What we can and can’t fix') %]</h2>
+
+<div class="embed-responsive embed-responsive-16by9">
+    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Uf0b_JAzpcY" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Shuold we use youtube-nocookie.com (or whatever it is) to avoid a cookie?

Reply:
Thanks for that, I didn't know it. I changed the embed so they use that one instead.